### PR TITLE
Add account change to luneClient

### PR DESCRIPTION
**Pull request by raguiar9080**
This makes it possible to both initialize a client with a particular
account being pointed to, or at any point in time simply change which
account is targeted.

Signed-off-by: Ruben Aguiar <r.aguiar9080@gmail.com>

### CODE REVIEW

File: src/templates/luneClient.hbs [hunk 1]
```diff
@@ -21,14 +21,24 @@ export class LuneClient {
     protected client: AxiosInstance
     protected config: ClientConfig
 
-    constructor(apiKey: string, baseUrl: string = 'https://api.lune.co', apiVersion: string = '1') {
+    constructor(
+        apiKey: string,
+        baseUrl: string = 'https://api.lune.co',
+        apiVersion: string = '1',
+        account: string,
```
Comment (jstasiak) [inline src/templates/luneClient.hbs:28]: I don't think we can have required parameters after optional/default-valued parameters

Comment (raguiar9080) [reply src/templates/luneClient.hbs:28]: Ups, this should be optional. Fixing 🙈

**Comment by jstasiak** [comment]
The selected account is not actually used anywhere at the moment, right?

**Comment by raguiar9080** [comment]
> The selected account is not actually used anywhere at the moment, right?

See the 'pair' PR on lune-ts: https://github.com/lune-climate/lune-ts/pull/14

**Review by jstasiak — APPROVED** [review]

**Pull request merged**